migrations/libs/json_keys.js

```diff
@@ -106,4 +106,5 @@ module.exports = {
     ItfLiquidatorUsdt: "ItfLiquidatorUsdt",
     ItfLiquidatorUsdc: "ItfLiquidatorUsdc",
     ItfLiquidatorDai: "ItfLiquidatorDai",
+    Multicall: "Multicall",
```

Comment:
Add migration scripts in Goerli and Mainnet which only setup this value - use appropriate address of that multical used on this specific environments